File: Modul-6/Soal 4/PRAK604-2410817120019-NadyaRamadhani.py
kode = input()
pesan = input()
kod = len(kode)
pes = len(pesan)
if kod != pes:
    print("Panjang kalimat berbeda, pesan palsu")
else:
    x=0
    y=0
    for i in range(len(kode)):
        if kode[i]==' ' and pesan[i]==' ':
            print(end=" ")
        elif kode[i]==pesan[i]:
            print("*", end=" ")
            x+=1
        else:
            print("#", end=" ")
            y+=1
    print(f"\n* = {x}")
    print(f"# = {y}")
    if x>=y:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")

File: Modul-6/Soal 4/PRAK604-2410817120019-NadyaRamadhani.c
#include <stdio.h>
#include <string.h>
int main(){
    char kode[30];
    char pesan[30];
    scanf("%[^\n]%*c", &kode);
    scanf("%[^\n]%*c", &pesan);
    int kod = strlen(kode);
    int pes = strlen(pesan);
    if(kod != pes){
        printf("Panjang kalimat berbeda, pesan palsu");
    }else{
        int x=0;
        int y=0;
        for(int i=0; i<kod; i++){
            if(kode[i]==' ' && pesan[i]==' '){
                printf(" ");
            }else if(kode[i]==pesan[i]){
                printf("*");
                x++;
            }else{
                printf("#");
                y++;
            }
        }
        printf("\n* = %d", x);
        printf("\n# = %d", y);
        if(x>=y){
            printf("\nPesan Asli");
        }else{
            printf("\nPesan Palsu");
        }
    }

    return 0;
}